ui/raidboss/data/06-ew/trial/rubicante-ex.ts

```diff
@@ -0,0 +1,243 @@
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Blazing Rapture',
+      type: 'StartsUsing',
+      netRegex: { id: '7D06', source: 'Rubicante', capture: false },
+      delaySeconds: 6,
+      response: Responses.bigAoe(),
+    },
+    {
+      id: 'RubicanteEx Inferno Solo Wing',
+      type: 'StartsUsing',
+      netRegex: { id: '7D0F', source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.protean!(),
+      outputStrings: {
+        protean: {
+          en: 'Protean',
+          de: 'Himmelsrichtung',
+          fr: 'Positions',
+          ja: '基本散会',
+          cn: '分散引导',
+          ko: '기본 산개',
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Scalding Signal',
+      type: 'StartsUsing',
+      netRegex: { id: '7D24', source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.outAndProtean!(),
+      outputStrings: {
+        outAndProtean: {
+          en: 'Out + Protean',
+          de: 'Raus + Himmelsrichtung',
+          fr: 'Extérieur + Positions',
+          ja: '外側 + 基本散会',
+          cn: '外侧 + 分散引导',
+          ko: '밖으로 + 기본 산개',
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Scalding Ring',
+      type: 'StartsUsing',
+      netRegex: { id: '7D25', source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.inAndProtean!(),
+      outputStrings: {
+        inAndProtean: {
+          en: 'In + Protean',
+          de: 'Rein + Himmelsrichtung',
+          fr: 'Intérieur + Positions',
+          ja: '内側 + 基本散会',
+          cn: '内侧 + 分散引导',
+          ko: '안으로 + 기본 산개',
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Sweeping Immolation / Partial',
+      type: 'StartsUsing',
+      netRegex: { id: '7D20', source: 'Rubicante', capture: false },
+      alertText: (_data, _matches, output) => output.spreadBehind!(),
+      outputStrings: {
+        spreadBehind: {
+          en: 'Spread behind Boss',
+          ja: 'ボスの後ろで散会',
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Sweeping Immolation / Total',
+      type: 'StartsUsing',
+      netRegex: { id: '7D21', source: 'Rubicante', capture: false },
+      infoText: (_data, _matches, output) => output.stackBehind!(),
+      outputStrings: {
+        stackBehind: {
+          en: 'Stack behind Boss',
+          ja: 'ボスの後ろで頭割り',
+        },
+      },
+    },
+    {
+      id: 'RubicanteEx Dualfire',
+      type: 'StartsUsing',
+      netRegex: { id: '7D2E', source: 'Rubicante', capture: false },
+      alertText: (data, _matches, output) => {
+        if (data.role === 'tank')
+          return output.tankCleave!();
+        return output.noTank!();
+      },
+      outputStrings: {
+        tankCleave: {
+          en: 'Dual Tank Cleave',
+          ja: 'MT/ST 同時にタンク範囲攻撃',
+        },
+        noTank: Outputs.avoidTankCleave,
+      },
+    },
+    /*
+    {
+      id: 'RubicanteEx Flamesent Shattering Heat Tether',
```

Comment:
I added this one but just a generic one-time "tank tethers" call when they come out, since there's extra tether lines after time it passes.